packages/twenty-front/src/modules/ui/field/input/components/TextAreaInput.tsx

```diff
@@ -36,12 +36,17 @@ const StyledTextArea = styled(TextareaAutosize)`
 `;
 
 const StyledTextAreaContainer = styled.div`
-  border: ${({ theme }) => `1px solid ${theme.border.color.medium}`};
+  border: ${({ theme }) => `1px solid ${theme.border.color.light}`};
   position: relative;
   width: 100%;
-  padding: ${({ theme }) => theme.spacing(2)} ${({ theme }) => theme.spacing(0)};
+  padding: ${({ theme }) => theme.spacing(2)} ${({ theme }) => theme.spacing(1)};
   border-radius: ${({ theme }) => theme.border.radius.sm};
   background: ${({ theme }) => theme.background.primary};
+  @media (max-width: 768px) {
```

Comment:
this does not feel right! Could you share more about what you are trying to achieve? Issue link or issue screenshot?

Reply:
issue link added in the description  

- if the screen size width is less than 768px a gap between placeholder and input field occurs
- screenshot before update
![367842746-7b447195-cbc0-4f0c-b708-9c3808f05975](https://github.com/user-attachments/assets/690389b8-2e8c-4bca-8650-d3125b8241a0)

- added a media query to fix the gap

-screenshot after update
<img width="768" alt="Screenshot 2024-10-13 at 10 05 53 PM" src="https://github.com/user-attachments/assets/befffd2c-0d06-4337-a073-37cf3c27fdc9">